includes/server/class-llms-rest-enrollments-controller.php

```diff
@@ -640,7 +642,7 @@ protected function build_collection_params() {
 			'validate_callback' => 'rest_validate_request_arg',
 		);
 
-		$query_params['context'] = 'view';
+		$query_params['context']['default'] = 'view';
```

Comment:
After a quick review it actually looks like this line could be removed (instead of updated) since the parent method already defines the default context (properly):

https://github.com/gocodebox/lifterlms-rest/blob/b2e4d63c6f739603c17c326609b9b06d3e82d750/includes/abstracts/class-llms-rest-controller.php#L114-L118

Reply:
Right you are!